.github/workflows/CI.yml

```diff
@@ -21,9 +21,15 @@ jobs:
     uses: ./.github/workflows/check_package.yml
     with:
       package-name: ${{ matrix.package-name }}
+  checks_succeeded:
+    name: Checks succeeded
+    needs: [check_package]
+    runs-on: ubuntu-latest
+    steps:
+      - run: exit 0
   test_package:
     name: Test ${{ matrix.package-name }}
-    needs: get_package_names
+    needs: [get_package_names]
```

Comment:
You don't need to change these `needs:` fields. You're allowed to use a string to specify a single dependency.